global-api/importer/mendoza_arg/load_mendoza_stationary_energy.sql

```diff
@@ -0,0 +1,37 @@
+/* Create a staging table */
```

Comment:
Please remove this file from the PR as it's not related to the emissions factors